uniffi_bindgen/src/interface/mod.rs

```diff
@@ -140,9 +141,9 @@ impl ComponentInterface {
         self.namespace.as_str()
     }
 
-    /// List the definitions for every Enum type in the interface.
-    pub fn iter_enum_definitions(&self) -> Vec<Enum> {
```

Comment:
heh - I was going to say I didn't hate the existing `iter_` prefix on these function names - until I got here and noticed they weren't iters at all :)

Reply:
Now that we have a new Askama that doesn't block us we could keep the `iter_` part of the method names and return iterators.  But changing these to return slices seems totally reasonable to me as well.